src/components/Tooltip/TooltipRootClose.js

```diff
@@ -40,31 +40,17 @@ class TooltipRootClose extends PureComponent {
     }
   }
 
-  addEventListeners() {
-    if (isNodeEnv()) { return }
-    document.addEventListener('click', this.handleMouseClick)
-    document.addEventListener('keyup', this.handleKeyUp)
-    window.addEventListener('resize', this.handleResize)
-  }
-
-  removeEventListeners() {
-    if (isNodeEnv()) { return }
-    document.removeEventListener('click', this.handleMouseClick)
-    document.removeEventListener('keyup', this.handleKeyUp)
-    window.removeEventListener('resize', this.handleResize)
-  }
-
-  handleMouseClick = (e) => {
+  handleMouseClick = e => {
```

Comment:
manual change: `sort-comp` rule wants `set/get` then `on`/`handle`  after the lifecycle methods